habitat/plan.sh

```diff
@@ -18,7 +19,7 @@ pkg_deps=(
   core/ruby26
   core/libxml2
   core/libxslt
-  core/libiconv
+  core/pkg-config
```

Comment:
I _think_ pkg-config can be only a build dependency in `pkg_build_deps`.

Reply:
Yeah that's probably true

Reply:
We've agreed to leave this here for now until we know its absence from runtime won't affect `chef gem install`.